src/sqliteformat.h

```diff
@@ -52,7 +51,7 @@ enum DBOperation {
   representation as Calendar and Incidences.
 */
 // exported just for unit test. Would be better to avoid.
-class MKCAL_EXPORT SqliteFormat
+class Q_DECL_EXPORT SqliteFormat
```

Comment:
This needs to be Q_DECL_EXPORT or Q_DECL_IMPORT depending on whether compiling the project or including the header in client app. 

See e.g.
https://github.com/sailfishos/libcommhistory/blob/master/src/libcommhistoryexport.h

So think we do still need the MKCAL_EXPORT but instead of low-level compiler attributes it can use the qt macros.

(Also while searching for project to refer to, noticed buteo lib is missing hide_symbols now).

Reply:
Ah ok, I understand now. I've reworked the incriminated commit accordingly.